interface-spec.md

```diff
@@ -66,15 +68,15 @@
 
 **Properties:**
 
-- `Node .graph` the named graph, which is an IRI, a BlankNode or NodeVariable.
+- `Node .why` the provenence, which may be a named node represenitng a document on the web which has this graph, or a BlankNode or NodeVariable. Also extednable for the results of infreence.
 
 ### DataFactory
 
 **Methods:**
 
-- `.iri(String iri)` returns a new instance of IRI.
-- `.blankNode()` returns a new instance of BlankNode.
+- `.sym(String iri)` returns a new instance of IRI.
+- `.bNode()` returns a new instance of BlankNode.
```

Comment:
Regarding RDF Interfaces, the goal of this Task Force is to come up with a JavaScript-idiomatic interface. RDF Interfaces can provide the building blocks (pointers to NamedNode discussions would be welcome), but other aspects are not straightforward to reuse (e.g., strong focus on synchronicity, whereas the single-threaded nature of JavaScript requires asynchronicity to execute certain tasks in a non-blocking way).